frontend/src/components/ViewTrips/trip.test.js

```diff
@@ -0,0 +1,24 @@
+import * as firebase from 'firebase/app';
+import 'firebase/firebase-firestore';
+
+import { getDateRange } from './trip.js'
+
+test('getDateRange test', () => {
+  const startMonth = 12;
+  const startDay = 17;
+  const startYear = 1995;
+  const endMonth = 5;
+  const endDay = 24;
+  const endYear = 1996;
```

Comment:
Could you clarify that these constants are the expected values and not the input values? It would be easier to read if I knew that as soon as I saw them.

Reply:
They are used for both. My thought process was that the dates should be the same for actual and expected values, so why not share the start/end date values that are used to create both the actual and expected strings.

Does this make sense? And if so, is that alright?

Reply:
Yes, that's fine. I left a comment on the code specifically addressing this as well.